doc/classes/String.xml

```diff
@@ -549,78 +617,79 @@
 			<param index="1" name="base" type="int" default="10" />
 			<param index="2" name="capitalize_hex" type="bool" default="false" />
 			<description>
-				Converts a unsigned [int] to a string representation of a number.
+				Converts the given unsigned [int] to a string representation, with the given [param base].
+				By default, [param base] is set to decimal ([code]10[/code]). Other common bases in programming include binary ([code]2[/code]), [url=https://en.wikipedia.org/wiki/Octal]octal[/url] ([code]8[/code]), hexadecimal ([code]16[/code]).
+				If [param capitalize_hex] is [code]true[/code], number representations after base 10 are returned in uppercase.
```

Comment:
Reading this broke my mind. May I suggest "digits higher than 9 will be represented in uppercase."?

Reply:
It's a bit misleading because the digits turn into letters, but the same time only letters can be either lowercase or uppercase, so the meaning can be inferred, absolutely.